### ISSUE TITLE
Building a Universe

### ISSUE THREAD

**Issue by lilyminium** [MDAnalysis/UserGuide#3]
**Story:** *As a* user, *I want* to concatenate selections of trajectories, *so that* I can use principal component analysis to compare them in a lower dimensional space. [Notebook example]

**Story:** *As a* user, *I want* to build my own Universe from scratch, *so that* [I can generate random molecules as test cases.](https://groups.google.com/forum/#!msg/mdnalysis-discussion/DCJzYzBbDJI/xUiqyAdfBQAJ) [Notebook example]

**Acceptance criteria:**

- [x] I understand how Universe.empty() works.
- [x] I understand how to load in coordinates manually.
- [x] I understand how to add and delete atoms/residues
- [x] I understand how to create a new Universe with Merge.
- [x] I have a link to the Topology section.

**Comment by richardjgowers** [comment]
Another relevant example here might be the `genconf`/`editconf` example of tiling out an existing Universe using `mda.Merge`/`U.empty()`

**Comment by orbeckst** [comment]
> As a user, I want to concatenate selections of trajectories,

Do you mean concatenating trajectories? If so, this means understanding what the ChainReader does.

**Comment by orbeckst** [comment]
The MemoryReader is also a good tool for these universe building hacks, see https://www.mdanalysis.org/mdanalysis/documentation_pages/coordinates/memory.html#creating-an-in-memory-trajectory-of-a-sub-system

**Comment by lilyminium** [comment]
> Do you mean concatenating trajectories? If so, this means understanding what the ChainReader does.

I meant being able to select and concatenate AtomGroups from different Universes. One common reason for this would be to perform principal component analysis to compare variance of different trajectories on a lower dimensional space. As far as I know, you can only do this by constructing a new Universe with Merge, and loading copied coordinates with MemoryReader.

e.g.
```python
def copy_coords(universe):
    
    def copy_pos(atom_group):
        return atom_group.positions.copy()
    
    selected = universe.select_atoms('protein')
    analysis = AnalysisFromFunction(copy_pos, selected)
    analysis.run()
    return analysis.results

pgp_coords = copy_coords(pgp)
pgp_atp_coords = copy_coords(pgp_atp)
pgp_atp_morphine_coords = copy_coords(pgp_atp_morphine)

combined_coords = np.vstack([pgp_coords, pgp_atp_coords, pgp_atp_morphine_coords])

combined = mda.Merge(pgp.select_atoms('protein'))
combined.load_new(combined_coords, format=MemoryReader)
```

**Comment by lilyminium** [comment]
Closed by #14 .
